Resources/Prototypes/Catalog/Cargo/cargo_fun.yml

```
@@ -1,23 +1,63 @@
 - type: cargoProduct
-  id: FunInstruments
+  id: FunInstrumentsVariety
   icon:
     sprite: Objects/Fun/Instruments/accordion.rsi
     state: icon
-  product: CrateFunInstruments
-  cost: 3000
+  product: CrateFunInstrumentsVariety
+  cost: 2000
   category: Fun
   group: market
 
 - type: cargoProduct
-  id: FunBrass
+  id: FunInstrumentsBrass
   icon:
     sprite: Objects/Fun/Instruments/structureinstruments.rsi
     state: tuba
-  product: CrateFunBrass
+  product: CrateFunInstrumentsBrass
   cost: 2500
   category: Fun
   group: market
 
+- type: cargoProduct
+  id: FunInstrumentsString
+  icon:
+    sprite: Objects\Fun\Instruments\bassguitar.rsi
```

My jackass copied the file path from the file explorer, and the slashes are backwards. I've only just realized now. Incredible
```suggestion
    sprite: Objects/Fun/Instruments/bassguitar.rsi
```